pkg/kudoctl/cmd/package_new.go

```diff
@@ -0,0 +1,147 @@
+package cmd
+
+import (
+	"errors"
+	"io"
+
+	"github.com/Masterminds/semver"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/cmd/generate"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/cmd/prompt"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
+	"github.com/kudobuilder/kudo/pkg/version"
+)
+
+const (
+	pkgNewDesc = `Create a new KUDO operator on the local filesystem`
+
+	pkgNewExample = `  # Create a new KUDO operator name foo 
+  kubectl kudo package foo
+`
+)
+
+type packageNewCmd struct {
+	name        string
+	out         io.Writer
+	fs          afero.Fs
+	interactive bool
+	overwrite   bool
+}
+
+// newPackageNewCmd creates an operator package on the file system
+func newPackageNewCmd(fs afero.Fs, out io.Writer) *cobra.Command {
+
+	pkg := &packageNewCmd{out: out, fs: fs}
+	cmd := &cobra.Command{
+		Use:     "new <operator name>",
+		Short:   "create new operator",
+		Long:    pkgNewDesc,
+		Example: pkgNewExample,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := validateOperatorArg(args); err != nil {
+				return err
+			}
+			pkg.name = args[0]
+			if err := pkg.run(); err != nil {
+				return err
+			}
+			return nil
+		},
+	}
+
+	f := cmd.Flags()
+	f.BoolVarP(&pkg.interactive, "interactive", "i", false, "Interactively create operator")
+	f.BoolVarP(&pkg.overwrite, "overwrite", "w", false, "overwrite existing directory and operator.yaml file")
+	return cmd
+}
+
+// run returns the errors associated with cmd env
+func (pkg *packageNewCmd) run() error {
+	folderDefault := "operator"
+	ovDefault := "0.1.0"
+	kudoDefault := version.Get().GitVersion
+	apiVerDefault := "kudo.dev/v1beta1"
```

Comment:
I find it strange to truncate word in between not truncated words. Let's do `apiVersionDefault` 🙏